examples/imagenet/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${CMAKE_BINARY_DIR}/include)
+
+IF(USE_CUDNN)
```

Comment:
since cpp verion of conv and pooling are merged. (lrn and batchnorm will be merged soon).
no need to check USE_CUDNN.